includes/payment-methods/class-upe-payment-method.php

```diff
@@ -74,6 +74,14 @@ abstract class UPE_Payment_Method {
 	 */
 	protected $icon_url;
 
+	/**
+	 * Supported customer locations for which charges for a payment method can be processed
+	 * Empty if all customer locations are supported
+	 *
+	 * @var string[]
+	 */
+	protected $countries = [];
```

Comment:
I'm open for discussion if we should be more specific in naming, like `supported_countries`